src/features/access-package/access_package_container.jsx

```diff
@@ -0,0 +1,41 @@
+import React, {useContext, useEffect} from "react";
+import {useDispatch, useSelector} from "react-redux";
+import AccessPackageList from "./access_package_list";
+import AccessPackageAdd from "./add/add_access_package";
+import {fetchComponents} from "../../data/redux/dispatchers/component";
+import {fetchEntities} from "../../data/redux/dispatchers/entity";
+import LoadingProgress from "../../common/status/LoadingProgress";
+import {fetchAccess} from "../../data/redux/dispatchers/access_package";
+import AppContext from "../../data/context/AppContext";
+import {fetchClients} from "../../data/redux/dispatchers/client";
+
+const AccessPackageContainer = () => {
+    const dispatch = useDispatch();
+    const componentConfiguration = useSelector(state => state.component_configuration.componentConfiguration);
+    const access = useSelector(state => state.access_package.accessPackages);
+    const context = useContext(AppContext);
+
+    useEffect(() => {
+            dispatch(fetchAccess(context.currentOrganisation.name));
+            dispatch(fetchComponents());
+            dispatch(fetchEntities());
+            dispatch(fetchClients(context.currentOrganisation.name));
+        }, [dispatch, context.currentOrganisation.name]
+    );
+
+    if (!componentConfiguration || componentConfiguration.length === 0 || !access) {
```

Comment:
Maybe put the if expression in a method to document it?

Reply:
Expression put in method